timescaledb/getting-started/create-hypertable.md

```diff
@@ -81,10 +73,21 @@ effective use of memory and disk resources. By automatically and transparently
 partitioning on time, hypertables improve resource use. Queries and
 data-stores become more efficient.
 
+   COMMENT:
+   I feel like we probably don't need this section but wherever this section is, 
+   I think it would be cool to include a graphic that shows these benefits 
+   https://iobeam.slack.com/archives/C0J94TE4F/p1646682667481189?thread_ts=1646682487.374619&cid=C0J94TE4F
+-->
+
 ## Learn more about hypertables and chunks
-See the [Hypertable How To](/how-to-guides/hypertables) to learn more about
-hypertables and best practices for configuring chunks.
+To learn more about hypertables and best practices for configuring chunks, see 
+[Hypertable How-To](/how-to-guides/hypertables). To learn more about why hypertables 
+help with storing and querying data, seethe [hypertables and chunks core concepts page][core-concepts-hypertables].
+
+## Next steps
+Next, ingest some sample stock trade data into TimescaleDB! Step 4, ['Add time-series data' section][add-data], 
+will show you how to populate the hypertable you just created. 
```

Comment:
We could probably do it as an admonition, since we already have the structure in place for that. We would just need design to come up with a suitable Eon icon for us, I expect. @jacobprall correct me if I'm wrong ;)